<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepDialog</class>
 <widget class="QDialog" name="StepDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="servoStep">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="delayStep">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay</string>
   </property>
  </widget>
  <widget class="QComboBox" name="output">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>65</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="endPosition">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>%</string>
   </property>
   <property name="minimum">
    <number>-150</number>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>End Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="time">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>166</width>
     <height>22</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StepDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
